ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -189,18 +207,22 @@ message UpdateUserResponse {
     User user = 1;
 }
 
-// Required authorization: ``HasRight(ParticipantAdmin)``
+// Required authorization: ``HasRight(ParticipantAdmin) OR IsAuthenticatedIdentityProviderAdmin(identity_provider_id)``
 message DeleteUserRequest {
     // The user to delete.
     // Required
     string user_id = 1;
+
+    // The id of the ``Identity Provider``
```

Comment:
`The id of an identity provider config`
?

Reply:
it's not a config id, it's actually an id of `Identity Provider`